src/flows/components/panel/FlowPanel.tsx

```diff
@@ -61,6 +64,49 @@ const FlowPanelHeader: FC<HeaderProps> = ({
     <div className="flow-panel--editable-title">Error</div>
   )
 
+  const printMap = useCallback(() => {
+    const stages = generateMap(true).reduce((acc, curr) => {
+      curr.instances.forEach(i => {
+        acc[i.id] = {
+          source: curr.text,
+          visualization: i.modifier,
+        }
+      })
+
+      return acc
+    }, {})
+
+    /* eslint-disable no-console */
+    flow.data.allIDs.forEach(i => {
```

Comment:
I know what this is and why this is here, but future me will be like, "the heck?" A comment on how this adds observability, etc... would be dope